ratelimit.go

```diff
@@ -171,6 +171,28 @@ func (tb *Bucket) takeAvailable(now time.Time, count int64) int64 {
 	return count
 }
 
+// Available returns the number of available tokens.
+// It will be negative when there are consumers waiting for tokens.
+// Note that this method is inherently racy - the returned count
```

Comment:
rather than "inherently racy" (since the available number is actually computed in a thread-safe way), maybe clarify that the returned number of available tokens does not guarantee calls like Take/TakeAvailable will succeed, and that this is primarily for introspection/metrics/reporting/debugging?